lib/MusicBrainz/Server/Controller/Admin/Attributes.pm

```diff
@@ -116,7 +116,17 @@ sub edit : Chained('attribute_base') Args(1) RequireAuth(account_admin) SecureFo
 sub delete : Chained('attribute_base') Args(1) RequireAuth(account_admin) SecureForm {
     my ($self, $c, $id) = @_;
     my $model = $c->stash->{model};
-    my $attr = $c->model($model)->get_by_id($id);
+    my $attr = $c->model($model)->get_by_id($id)
+        or $c->detach(
+            '/error_404',
+            [ l(
+                'Found no attribute of type “{model}” with ID “{id}”.',
+                {
+                    id => $id,
+                    model => $model,
+                },
+            ) ],
```

Comment:
This message is for admin only so it doesn’t need to be translated. **Untested** suggestion:

```suggestion
            [ "Found no attribute of type “$model” with ID “$id”." ],
```

Reply:
It works fine as long as we also `use utf8;` :) Will do that.